locales-pending/settings-premium.ftl

```diff
@@ -63,3 +63,97 @@ settings-delete-monitor-plus-account-dialog-cancel-button-label = Never mind, ta
 ## Monthly Monitor Report
 
 settings-alert-preferences-allow-monthly-monitor-plus-report-title = Monthly { -brand-monitor-plus } report
+
+## Update scan info
+
+settings-update-scan-info-description = { -brand-monitor } is most effective at protecting your info when you add specific details. Add any of your name variations, emails, or locations. <a>Why should I add details for my scan?</a>
+settings-update-scan-info-upsell-cta-label = Upgrade to { -brand-monitor-plus } to protect your personal info
+settings-details-saved-notification-title = Details saved
+settings-details-saved-notification-message = Thanks for helping { -brand-monitor } protect you. We’ll use this info in future scans.
+
+## Details about you
+
+settings-details-about-you-header = Details about you
+settings-details-about-you-description = { -brand-monitor } will use this info for scans and data removal requests.
+settings-details-about-you-name-label = Name
+settings-details-about-you-date-of-birth-label = Date of birth
+settings-details-about-you-phone-label = Phone number
+settings-details-about-you-location-label = Location
+settings-details-about-you-cta-label = Add details
+# Variables:
+#   $moreCount (number) - The additional number of details for this
+settings-details-about-you-more-indicator = +{ $moreCount } more
+
+## Email addresses
+
+settings-email-addresses-header = Email addresses
+settings-email-addresses-description = { -brand-monitor } will alert you if these emails show up in known breaches.
+settings-email-addresses-add-email-cta = Add email address
+# Variables:
+#   $limit (number) - Number of emails that can be added for monitoring.
+settings-email-addresses-add-email-indicator-limit = Add up to { $limit }
+settings-email-addresses-add-email-resend-button-label = Resend verification link
+settings-email-addresses-add-email-remove-button-label = Remove
+
+## Email address dialog
+
+settings-email-addresses-initial-dialog-header = Add an email address
+settings-email-addresses-initial-dialog-description = We’ll send a verification link for you to confirm you’d like to include in a future { -brand-monitor } scan.
+settings-email-addresses-initial-dialog-add-email-input-label = Enter email address
+settings-email-addresses-initial-dialog-add-email-cta = Send verification link
+# Variables:
+#   $email (string) - Email address the confirmation has been sent to.
+settings-email-addresses-confirmation-dialog-header = Verification link sent to <b>{ $email }</b>
+settings-email-addresses-confirmation-dialog-description = We’ll send a verification link for you to confirm you’d like to include in a future { -brand-monitor } scan.
```

Comment:
Same question here: "to include in" seems to be missing an object.

Reply:
This string is actually not supposed to be a duplicate and I’ve added the correct sting in https://github.com/mozilla/blurts-server/pull/5793/commits/338a0fd0bc12d6d437bc410e39e70b98103e7b41.